docs/_integration-with-exisiting-apps-swift.md

```diff
@@ -100,34 +100,48 @@ The `Podfile` will contain a boilerplate setup that you will tweak for your inte
 Ultimately, your `Podfile` should look something similar to this:
 
 ```
-source 'https://github.com/CocoaPods/Specs.git'
-
-# Required for Swift apps
-platform :ios, '8.0'
-use_frameworks!
-
-# The target name is most likely the name of your project.
-target 'swift-2048' do
-
-  # Your 'node_modules' directory is probably in the root of your project,
-  # but if not, adjust the `:path` accordingly
-  pod 'React', :path => '../node_modules/react-native', :subspecs => [
-    'Core',
-    'CxxBridge', # Include this for RN >= 0.47
-    'DevSupport', # Include this to enable In-App Devmenu if RN >= 0.43
-    'RCTText',
-    'RCTNetwork',
-    'RCTWebSocket', # needed for debugging
-    # Add any other subspecs you want to use in your project
-  ]
-  # Explicitly include Yoga if you are using RN >= 0.42.0
-  pod "Yoga", :path => "../node_modules/react-native/ReactCommon/yoga"
-
-  # Third party deps podspec link
-  pod 'DoubleConversion', :podspec => '../node_modules/react-native/third-party-podspecs/DoubleConversion.podspec'
-  pod 'glog', :podspec => '../node_modules/react-native/third-party-podspecs/glog.podspec'
-  pod 'Folly', :podspec => '../node_modules/react-native/third-party-podspecs/Folly.podspec'
+require_relative '../node_modules/react-native/scripts/react_native_pods'
+require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
+
+platform :ios, '12.4'
+install! 'cocoapods', :deterministic_uuids => false
```

Comment:
We are updating this in the template. the new setup would be:
```ruby
platform :ios, min_ios_version_supported
prepare_react_native_project!
```

Perhaps it could make sense to link to the [most recent `Podfile`](https://github.com/facebook/react-native/blob/main/template/ios/Podfile) we have in the `template`, rather than manually copy and paste it?
So we don't have to come back to this if we make updates, what do you think?

Reply:
included the link to the Podfile Template.